contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -511,6 +511,86 @@ <h4 class="modal-title" id="confirmDeleteTitle">Delete Contribution</h4>
             document.getElementById("myDropdown").classList.toggle("show");
         }
     })
+    /*
+    // Custom dropdown menu functions
```

Comment:
Please remove the commented-out code. Thanks.

Reply:
"The javascript needed for the full customization is commented out. The other related styling classes is also present in the style.css file. These can be deleted once the overall view is finalized."
Without that javascript code, the full customization is not acheivable.

Reply:
removed commented out javascript and extra stylings